jdaviz/configs/imviz/plugins/links_control/links_control.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <v-card flat tile>
+    <v-card>
+      <v-card-text>
+        <v-container>
+          <v-row>
+            <v-col>
+              <v-radio-group 
+                label="Link type"
+                hint="Type of linking to be done."
+                v-model="link_type"
+                persistent-hint
+                row>
+                <v-radio
+                  v-for="item in link_types"
+                  :key="item"
+                  :label="item"
+                  :value="item"
+                ></v-radio>
+               </v-radio-group>
+            </v-col>
+          </v-row>
+          <v-row v-if="false">
+            <v-col>
+              <v-switch
+                label="Fallback on Pixels"
+                hint="If WCS linking fails, fallback to linking by pixels."
+                v-model="wcs_use_fallback"
+                persistent-hint
+                inset>

```

Comment:
```suggestion
                persistent-hint>
```

I noticed since adding the switch that all other switches in the UI do not use the inset styling - removing this would then be consistent with the rest.